packages/design-system/src/atoms/buttonV2/index.tsx

```diff
@@ -101,10 +79,35 @@ const BaseButton = ({
     },
   })
 
+  const paddingY = useMemo(() => {
+    if (isActive)
+      return paddingVertical - (active.borderWidth - rest.borderWidth)
+    if (isHover) return paddingVertical - (hover.borderWidth - rest.borderWidth)
+    return paddingVertical
```

Comment:
Let's tend to use if blocks with brackets, please. It improves readability a lot.

```suggestion
    if (isActive) {
      return paddingVertical - (active.borderWidth - rest.borderWidth)
    }
    if (isHover) {
      return paddingVertical - (hover.borderWidth - rest.borderWidth)
    }
    return paddingVertical
```